core/lib/spree/permission_sets/default_customer.rb

```diff
@@ -23,6 +23,7 @@ def activate!
         can :display, StockLocation, active: true
         can :display, Taxon
         can :display, Taxonomy
+        can [:display_address_book, :update_address_book, :remove_from_address_book], Spree.user_class, id: user.id
```

Comment:
I do think that these custom permissions are the correct approach. Could we change :update_address_book to :save_in_address_book so it follows the method convention like remove_from_address_book does? Just a nitpick, not too opinionated on it.